src/vs/workbench/contrib/externalTerminal/browser/externalTerminal.contribution.ts

```diff
@@ -20,119 +20,147 @@ import { IRemoteAgentService } from 'vs/workbench/services/remote/common/remoteA
 import { ContextKeyExpr } from 'vs/platform/contextkey/common/contextkey';
 import { IWorkbenchContribution, IWorkbenchContributionsRegistry, Extensions as WorkbenchExtensions } from 'vs/workbench/common/contributions';
 import { Disposable } from 'vs/base/common/lifecycle';
-import { isWeb, isWindows } from 'vs/base/common/platform';
+import { isWindows } from 'vs/base/common/platform';
 import { dirname, basename } from 'vs/base/common/path';
 import { LifecyclePhase } from 'vs/workbench/services/lifecycle/common/lifecycle';
 import { Registry } from 'vs/platform/registry/common/platform';
 import { IExternalTerminalConfiguration, IExternalTerminalService } from 'vs/platform/externalTerminal/common/externalTerminal';
 import { TerminalLocation } from 'vs/platform/terminal/common/terminal';
 
 const OPEN_IN_TERMINAL_COMMAND_ID = 'openInTerminal';
-CommandsRegistry.registerCommand({
-	id: OPEN_IN_TERMINAL_COMMAND_ID,
-	handler: async (accessor, resource: URI) => {
-		const configurationService = accessor.get(IConfigurationService);
-		const editorService = accessor.get(IEditorService);
-		const fileService = accessor.get(IFileService);
-		const integratedTerminalService = accessor.get(IIntegratedTerminalService);
-		const remoteAgentService = accessor.get(IRemoteAgentService);
-		const terminalGroupService = accessor.get(ITerminalGroupService);
-		let externalTerminalService: IExternalTerminalService | undefined = undefined;
-		try {
-			externalTerminalService = accessor.get(IExternalTerminalService);
-		} catch {
-		}
+const OPEN_IN_INTEGRATED_TERMINAL_COMMAND_ID = 'openInIntegratedTerminal';
```

Comment:
hi @weartist 

I see you created new `OPEN_IN_INTEGRATED_TERMINAL_COMMAND_ID`, when Integrated is default behaviour, this should create new `OPEN_IN_EXTERNAL_TERMINAL_COMMAND_ID` instead of.

I think this merge lead to issue https://github.com/microsoft/vscode/issues/182602
Could you please check again?